rules/falco_rules.yaml

```diff
@@ -172,6 +172,9 @@
 - list: gitlab_binaries
   items: [gitlab-shell, gitlab-mon, gitlab-runner-b, git]
 
+- list: dynatrace_binaries
```

Comment:
how bad it is if we don't exclude `dynatrace` here? I am still debating with myself whether we should include the security vendors in the open source version.

Reply:
I don’t think it should be in the public repo. We can recommend this to our customers personally.